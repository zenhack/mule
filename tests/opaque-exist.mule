#XFAIL: type formatting
{} : { type t }
