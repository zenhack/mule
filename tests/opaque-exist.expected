
Desugared:

	(: (record (types ()) (values ()))
	 (exist $12 (record (types ((t $12))) (values ()))))

inferred type:

	(exist a (record (types ((t a))) (values ())))

Runtime term:

	(Record ())

Evaluated:

	(Record ())
